### PR TITLE
Chore/134346377/host on heroku

### PR DESCRIPTION
**Pull request by andela-oaladeusi**
#### What does this PR do?  
Host project on Heroku    
         
#### Description of Task to be completed?    
This is important to build, run, and operate the application entirely in the cloud.     
           
#### How should this be manually tested?       
https://banaskotest.herokuapp.com/#!/

### CODE REVIEW

File: config/express.js [hunk 1]
```diff
@@ -2,7 +2,9 @@
  * Module dependencies.
  */
 var express = require('express'),
-    mongoStore = require('connect-mongo')(express),
+    session=require('express-session'),
```
Comment (adebayo-maborukoje) [inline config/express.js]: *NIT PICK*
space is required between the equality sign and the `require`

File: config/env/all.js [hunk 2]
```diff
@@ -3,10 +3,10 @@
...
 var keys = rootPath + '/keys.txt';
-var testLocal = 'mongodb://localhost/cfh';
+var testLocal = 'mongodb://localhost/cft';
 
 module.exports = {
   root: rootPath,
-  port: process.env.PORT || 3000,
-  db: process.env.MONGOHQ_URL || testLocal
+  port: process.env.PORT || 4000,
+  db: process.env.MONGOHQ_URL ||testLocal
```
Comment (adebayo-maborukoje) [inline config/env/all.js]: Why did you remove the space here?

File: public/js/services/game.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ angular.module('mean.system')
     table: [],
     czar: null,
     playerMinLimit: 3,
-    playerMaxLimit: 6,
+    playerMaxLimit: 11,
```
Comment (adebayo-maborukoje) [inline public/js/services/game.js]: I'm not sure this should be in this PR

**Comment by andela-kakpobome** [comment]
This PR title did not follow the convention in the wiki, why was it merged? @adebayo-maborukoje @andela-oaladeusi
